pkg/windows/windows.go

```diff
@@ -930,9 +930,11 @@ func (vm *windows) isContainersFeatureEnabled() (bool, error) {
 	return strings.Contains(out, "Enabled"), nil
 }
 
-// isWindowsDefenderEnabled returns true if the Windows Defender antivirus/firewall is installed on the Windows instance
-func (vm *windows) isWindowsDefenderEnabled() (bool, error) {
-	command := "(Get-Service | where {$_.DisplayName -Like 'Windows Defender*'}).Count -gt 0"
+// isWindowsDefenderRunning returns true if the Windows Defender antivirus/firewall is enabled on the Windows instance
```

Comment:
```suggestion
// isWindowsDefenderRunning returns true if the Windows Defender antivirus/firewall is running on the Windows instance
```

Reply:
Enabled == running according to [MS docs](https://learn.microsoft.com/en-us/microsoft-365/security/defender-endpoint/microsoft-defender-antivirus-compatibility?view=o365-worldwide#use-windows-powershell-to-confirm-that-microsoft-defender-antivirus-is-running)

> Use Windows PowerShell to confirm that Microsoft Defender Antivirus is running
> - You should see MsMpEng.exe if Microsoft Defender Antivirus is enabled.

Reply:
I can see Windows Defender being enabled i.e. the service is set to start but not auto-restart and the process has crashed and hence is not running. 

PS: every pattern that you see in "upstream" or other documentation need not be a good pattern and we should use our judgement.

Reply:
I'll re-word to consistently use `running`